### DISCUSSION TITLE
How to create a mixed grain ensemble ?

### DISCUSSION THREAD

**Discussion by ppothana** [neperfepx/neper#462] [Q&A] [answered]
I would like to generate tesselation of mixed grain ensemble as shown in figure. Could you please help me with the commands to be used to generate such mixed structures ?
![image](https://user-images.githubusercontent.com/104273832/166605260-248ced66-5883-4c8b-a72c-5fe9ae825d7c.png)

Figure source : (Quey et al., 2011) https://hal.archives-ouvertes.fr/hal-00858028

**Comment by rquey** [comment, marked as answer]
Such a structure is obtained when there are no seeds on the right side of the domain. It can for example be generated as follows:

1. Generate seed coordinate in cube(0.6,1,1):
```
neper -T -n 1000 -domain "cube(0.6,1,1)" -statseed x,y,z
```
2. Generate tessellation is cube(1,1,1) from these seeds:
```
neper -T -n 1000 -domain "cube(1,1,1)" -morphooptiini "coo:file(n1000-id1.stseed)"
```
3. Visualize:
```
neper -V n1000-id1.tess -showcell "y>0.5" -showedge "cell_shown||domtype==1" -print img
```
![image](https://user-images.githubusercontent.com/2659376/166633495-c12873e2-18fd-4678-87c6-587326298a8c.png)

**Reply by ppothana** [reply]
Thank you very much.
